docs/documentation/overview.md

```diff
@@ -97,7 +97,7 @@ To learn more about Zulip's general user documentation,
 ### Integrations documentation
 
 To learn more about Zulip's integrations documentation,
-[visit it on zulip.com](https://zulip.com/integrations/) or
+[visit it on zulip.com](https://zulip.com/api/integrations-overview) or
 [read our guide on writing user documentation](integrations.md).
```

Comment:
I think we probably want to link to both -- one as "how it is displayed" and the second as what integrations are.